azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/implementation/LoadBalancerImpl.java

```diff
@@ -401,9 +431,40 @@ public ProbeImpl defineHttpProbe(String name) {
         return new ProbeImpl(name, inner, this);
     }
 
+    @Override
+    public LoadBalancerImpl withoutProbe(String name) {
+        if (this.httpProbes.containsKey(name)) {
+            this.httpProbes.remove(name);
+        } else if (this.tcpProbes.containsKey(name)) {
+            this.tcpProbes.remove(name);
+        }
+
+        List<ProbeInner> probes = this.inner().probes();
+        if (probes != null) {
+            for (int i = 0; i < probes.size(); i++) {
+                if (probes.get(i).name().equalsIgnoreCase(name)) {
+                    probes.remove(i);
+                    break;
+                }
+            }
+        }
+
+        return this;
```

Comment:
Does server allows an http and a tcp probe with same name? if no please ignore this comment else we might want to remove the correct one in the loop based on from which list we removed the probe [`this.httpProbes` or `this.tcpProbes`]